common/src/warp_runner/manager/commands/constellation_commands.rs

```diff
@@ -395,6 +401,29 @@ async fn upload_files(
         };
         let local_path = Path::new(&file_path).to_string_lossy().to_string();
 
+        let file_size = match fs::metadata(&local_path) {
```

Comment:
Since this is a async function, this call would be blocking so we should use the call from tokio

```suggestion
        let file_size = match tokio::fs::metadata(&local_path).await {
```